docs/migrating-from-legacy-packages.md

```diff
@@ -0,0 +1,606 @@
+---
+id: migrating-from-legacy-packages
+title: Migrating from Legacy Happo Packages
+---
+
+This guide will help you migrate from the legacy Happo packages (`happo.io`,
+`happo-cypress`, `happo-playwright`, etc.) to the new unified `happo` package.
+
+## Overview
+
+The new `happo` package is a modern, TypeScript-first rewrite that consolidates
+functionality from multiple legacy packages into a single, unified package. This
+migration brings several benefits:
+
+- **Unified package**: One package instead of multiple (`happo.io`,
+  `happo-cypress`, `happo-playwright`, `happo-plugin-storybook`, `happo-e2e`,
+  and `happo-static`)
+- **TypeScript support**: Built-in TypeScript support with full type definitions
+- **Modern ES modules**: Uses ES modules
+- **Improved configuration**: More flexible configuration file formats
+- **Simplified target configuration**: Plain object syntax instead of
+  class-based targets
+
+## Breaking Changes
+
+### Node.js Version Requirement
+
+The new `happo` package requires **Node.js >= 22**. Ensure your environment
+meets this requirement before migrating.
+
+### Package Installation
+
+Remove all legacy Happo packages and install the new unified `happo` package:
+
+```bash
+npm uninstall happo.io \
+  happo-cypress \
+	happo-playwright \
+	happo-e2e \
+	happo-plugin-storybook \
+	happo-static
+npm install --save-dev happo
+```
+
+The new `happo` package replaces all of these legacy packages. All functionality
+is now included in the single unified package.
+
+### Configuration File Format
+
+The new package supports multiple configuration file formats and encourages
+TypeScript/ES modules:
+
+**Before (legacy):**
+
+```js title=.happo.js
+const { RemoteBrowserTarget } = require('happo.io');
+
+module.exports = {
+  apiKey: process.env.HAPPO_API_KEY,
+  apiSecret: process.env.HAPPO_API_SECRET,
+  targets: {
+    'chrome-desktop': new RemoteBrowserTarget('chrome', {
+      viewport: '1024x768',
+    }),
+  },
+};
+```
+
+**After (new):**
+
+```js title=happo.config.ts
+import { defineConfig } from 'happo';
+
+export default defineConfig({
+  apiKey: process.env.HAPPO_API_KEY!,
+  apiSecret: process.env.HAPPO_API_SECRET!,
+
+	project: 'default',
+
+	integration: {
+		type: 'storybook',
+		configDir: '.storybook',
+	},
+
+  targets: {
+    'chrome-desktop': {
+      browserType: 'chrome',
+      viewport: '1024x768',
+    },
+  },
+});
+```
+
+### Target Configuration
+
+One of the most significant changes is how targets are defined. The new package
```

Comment:
I'm not sure this statement is true. All we do is remove the need for `new RemoteBrowserTarget(...)`